framework/src/org/checkerframework/common/value/qual/ArrayLenRange.java

```diff
@@ -7,9 +7,9 @@
 import org.checkerframework.framework.qual.SubtypeOf;
```

Comment:
See comment on `ArrayLen` about renaming. This name should just be change to *Range, where * is whatever we rename `ArrayLen` to.